### DISCUSSION TITLE
Having issues with restart of the comfyUI for using manager

### DISCUSSION THREAD

**Discussion by KhushShah19** [Comfy-Org/ComfyUI-Manager#142] [Q&A]
I am using ComfyUI on the server (runpod ), I can't find the way to restart the UI (program). 
I also tried to run the main.py file but it got some weird error. 
Need some help with this

**Comment by ltdrdata** [comment]
This is not something that can be handled by the Manager. 
Let's hope that this [PR](https://github.com/comfyanonymous/ComfyUI/pull/1814) gets merged.
